### PR TITLE
Tabs Created

### LINKED ISSUE #3: Create Tabs
Homepage 
Meet the Revolutionaries 
Support
FAQs
Press
Contact
Blog

Content Here: https://drive.google.com/drive/folders/0BwpCIeAR0xFcTVlXc1FvQXZvcHM?usp=sharing

### PR DESCRIPTION
**Pull request by amritsrivastava**
closes #3

## What all changes are included in this PR?
I have created tabs for all the items listed in the issue and created a basic header.

## Is there anything left to be done in the addressed issue? If yes, then state the remaining tasks here.
Nothing left.
Kindly review my PR and suggest changes.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App.js';
```
Comment (lalitmee) [inline src/index.js]: There is no need of adding the file extension while importing that file. It was correct before. :+1:

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by lalitmee** [comment]
Hey @amritsrivastava, you only need to include the file extension when you are importing **CSS** file otherwise you are not required to add the file extension in the case of **JS** file. Please do check your files wherever you have done this. 

One more thing is that the style to Header tabs is not applied. If you can improve that style, it would be great.

I am seeing like the below image:
screenshot from 2018-06-29 10-51-24

**Comment by amritsrivastava** [comment]
Did your internet connection is turned on while you run the page, as the header I used is of Bootstrap so it might possible that due to lack of internet connection the bootstrap's css and js scripts won't load and you are getting this output.

**Comment by lalitmee** [comment]
Actually, I have a very high-speed internet connection. :wink:

**Comment by lalitmee** [comment]
Yeah, it's fine. I saw the design. Great. :+1:

**Comment by vaarigupta** [comment]
@amritsrivastava  I have cloned your repo and run in my local machine by using the command " npm start" , then its showing me an error some node modules are missing ? could you help me out to run your code in my Pc?

**Comment by lalitmee** [comment]
Hey @vaarigupta, actually @amritsrivastava, is having the main code for these tabs on a patch branch. So I would like to give you a suggestion that you don't need to work in your local machine if you don't want to. There is an online platform [CodeSandbox](https://codesandbox.io). What you need to do is only copy any repository url and paste it into the new codesandbox and your project will run. 

For example: https://codesandbox.io/s/github/amritsrivastava/Kranti/tree/amrit-patch-1

I have copied the url of repo of @amritsrivastava. :+1: :innocent: 

I hope it helps.

**Comment by vaarigupta** [comment]
Thanks a lot @lalitmee  .You cleared my Doubt . I was working on this but could not figure out . Ok so I will create tabs for other pages except homepage

**Comment by vaarigupta** [comment]
The **points that I want to Sum up** about tabs  ->

1.   Please check out this [link]( https://casefoundation.org/) We can get some great idea . I really liked its UI .There's a picture in the background and when we scroll then the contents move up and navbar remains fixed .Moreover , there is fixed content on the picture .
2. Next Go through this [link ->](http://www.mahelerecen.org.in/?utm_source=google&utm_medium=cpc&utm_campaign=simplified%20campaign&utm_term=NGO%20and%20NPO) . This is wonderful .There's is a variety of pictures on the entire homepage which keeps on changing while scrolling the page plus the content pop ups on the screen and some of the content is fixed on the images which makes it damn fascinating to the user and make its UI astonishing .
3 . Have a look on this [link](https://www.convoyofhope.org/) . The hovering Effect of navbar and one more point to grab is the addition of video in the background for more eye-catchy look.

**Comment by lalitmee** [comment]
Hello @guys, I think on the homepage some of the tabs should be in the footer like **Support**, **Press**, **Contact**. Tabs which should be emphasized more should be in the header. 

Hey @amritsrivastava, what do you think? :innocent:

**Comment by amritsrivastava** [comment]
@vaarigupta I liked the header of first and third links. Both are pretty cool we can also convert our header like first one by just making it's background transparent. 
@lalitmee I absolutely agree with you for inserting some tabs in the footer.

**Review by AvaniVerma — COMMENTED** [review]
Overall, it's great. However, consider the following 

1. Check the warning message in console. There's a body tag in div.

2. It's not entirely responsive between the width range 771 and 990

3. Increase the font size of 'Kranti' to make it significantly bigger than that of the nav contents, especially on bigger screen.

4. Make the logo a link that redirects to the home page.

**Comment by lalitmee** [comment]
Hey @amritsrivastava, are you considering the changes which @AvaniVerma mentioned? :thinking:

**Comment by amritsrivastava** [comment]
yes, I will do those changes as soon as possible.

**Pull request merged**